templates/base.html.twig

```diff
@@ -3,6 +3,9 @@
    common elements and decorates all the other templates.
    See https://symfony.com/doc/current/templates.html#template-inheritance-and-layouts
 #}
+
+{% set _route = app.request.get('_route') %}
```

Comment:
you should define this variable only in the `header` block where you need it rather than at the root of the layout (where it would impact the scope of all rendered templates)

Reply:
Ok, I see that. Thanks.